components/table/src/table/tr.directive.ts

```diff
@@ -11,7 +11,7 @@ import { NzThMeasureDirective } from '../cell/th-measure.directive';
 import { NzTableStyleService } from '../table-style.service';
 
 @Directive({
-  selector: 'tr:not([mat-row]):not([mat-header-row]):not([nz-table-measure-row]):not([nzExpand]):not([nz-table-fixed-row])',
+  selector: 'tr:not([mat-row]):not([mat-header-row]):not([nz-table-measure-row]):not([nzExpand]):not([nz-table-fixed-row]):not(tfoot tr)',
```

Comment:
Hi, `:not(tfoot tr)` won't work in angular selector, would it be ok to remove this selector?

Reply:
![image](https://user-images.githubusercontent.com/7550366/107341005-6c38e100-6af9-11eb-8d51-d4b8f630ff31.png)

In ant design react, `tr` in `tfoot` cannot add the `ant-table-summary` class

Reply:
I would like to suggest add`isInsideTfoot` param like `isInsideTable`

Reply:
I have no idea... Should I add a service to recognize if it is inside tfoot?

Reply:
you can add a `tfoot` directive and inject it inside the `tr` directive

Reply:
I have fixed it.